src/server/rest/v1/service/UserService.ts

```diff
@@ -232,6 +232,11 @@ export default class UserService {
         adminData.plateID = filteredRequest.plateID || null;
         await UserStorage.saveUserAdminData(req.tenant, user.id, adminData);
       }
+      if (Utils.objectHasProperty(filteredRequest, 'technical')) {
```

Comment:
saveUserAdminData should be called once for technical and plateID as you did it in createUser :-( !

Reply:
user.projectedFields can also be used to replace the IF statements:
`if (Authorizations.isAdmin(req.user) || Authorizations.isSuperAdmin(req.user)) {`